### DISCUSSION TITLE
Files not moved!

### DISCUSSION THREAD

**Discussion by hamaryns** [beetbox/beets#4638] [Q&A]
Hi all,
Looks like I’m doing something wrong, or is it beets?  I started an import, recognized a file, apply, on to next, then abort.  But the file is not physically moved to my library.  It is in the database though, says beet ls.  beet ls -p says it is in the folder I just imported it from, but I have import: move: yes in my config file.

Before, with exactly the same command and configuration, files of other subdirectories in the folder I’m importing, were moved fine.

Would be glad if somebody could shine a light on this.

Here’s some logs.

TIA, H.
**command line:**
[beet7.log](https://github.com/beetbox/beets/files/10433752/beet7.log)
**logs:**
[beet8.log](https://github.com/beetbox/beets/files/10433753/beet8.log)
**config:**
directory: /media/hendrik2/Scharnier/Muziek
library: ~/.beets/musiclibrary.db
import:
    move: yes
    duplicate_action: merge
log: beetslog.txt
art_filename: $albumartist – $album
plugins: fetchart lastgenre chroma fromfilename mbsync
paths:
    default: Albums/$albumartist/$album%aunique{}/$artist – $title
    singleton: Losse nummers/$artist/$artist – $title
    comp: Compilaties/$album%aunique{}/$artist – $title
acoustid:
    apikey: uqolV41Xwc

**Comment by hamaryns** [comment]
I just tried another folder with just one album, that worked as expected.  (Imported and moved)

**Comment by sampsyo** [comment]
Hi there! I noticed you mentioned that you aborted the import. If so, this is sort of the expected behavior—beets tries to shut down the import pipeline as quickly as possible, so if there are files still pending to be copied in the background (the importer is multithreaded, so multiple albums can be "in flight" at the same time), then that copy operation will also be aborted. Would this apply to your situation?

**Comment by hamaryns** [comment]
OK, I thought that aborting meant that all tracks/albums that were accorded still were moved, that it finished what it started, so to say.  How else to stop a long import to continue later?

Anyway, even when not aborting, but waiting it still does not move the file.  It has been open for at least 30 min. now, I just do not respond to the terminal, but the files are still there.  So that does not seem to have been the problem.
